crawl_jobs/spiders/pyjobs.py

```diff
@@ -18,7 +17,12 @@ def parse(self, response):
         yield from response.follow_all(pagination_links, self.parse)
 
     def parse_jobs(self, response):
+        description = response.xpath('//div[@class="container"]//div[@class="row"]//text()')[44:].getall()
+        clear_description = [i.strip() for i in description]
+        description_to_sanitize = [i for i in clear_description if i != ""] 
```

Comment:
O que acha de?
```suggestion
        description_sanitized = [i.strip() for i in description if i.strip()]
```
Voce nao precisa fazer `if i != ""` se voce quer satisfazer que existe conteudo numa string. 
bastaria ver se ha o conteudo como nesse if sugerido.